debian/jessie/foreman/control

```diff
@@ -3,7 +3,7 @@ Maintainer: Greg Sutcliffe <greg.sutcliffe@gmail.com>
 Section: web
 Priority: extra
 Standards-Version: 3.9.1
-Build-Depends: debhelper (>= 7), cdbs, bundler (>= 1.1), ruby2.1-dev, git | git-core,
+Build-Depends: debhelper (>= 7), cdbs, bundler (>= 1.3), ruby2.1-dev, git | git-core,
```

Comment:
This is to explicitly state that bundler >= 1.3 is _needed_.

For jessie and trusty, no bundler packages were added in this PR and the ones coming from the stock OK repos were/are already used with success.